code/physics/physics.h

```diff
@@ -100,6 +100,7 @@ typedef struct physics_info {
 
 	matrix ai_desired_orient;   // Asteroth - This is only set to something other than the zero matrix if Framerate_independent_turning is enabled, and 
 								// only by the AI after calls to angular_move. It is read and then zeroed out for the rest of the frame by physics_sim_rot
+	vec3d acceleration;
```

Comment:
I would call this `derived_acceleration` or something to make it clear that this is calculated based on the effects of whatever the physics code happens to be doing, *not* instructions for the physics code.

Reply:
That's a very good point, but I do prefer the conciseness of `acceleration`. Would a comment that makes this clear suffice? Thinking about it, it follows more or less how `velocity` works, at least for fighters, for better or for worse, in that its really more of a description of the trend of its position, then necessarily a predictor of it (even if it is correlated to it)

Reply:
If this is consistent with the existing code, then fair enough.  But a comment would be helpful.